helm/steps-to-deploy/READme.md

```diff
@@ -0,0 +1,35 @@
+## Steps to Deploy ERP into K8s Cluster
+
+1. Clone git repository for helm chart: https://github.com/truthfool/openg2p-erp-docker.git
+
+
+2. Create persistent volume claim in cloud using :
+`kubectl apply -f pvc.yaml`
+
+
+3. Deploy odoo and postgresql pod using :
+ `helm install openg2p -f values.yaml bitnami/odoo`
+
+
+4. Using k9s enter into openg2p-postgresql pod and grant superuser access to openg2p role.
+
+5. Login to postgres user using:
+
+`psql -h localhost -d openg2p -U postgres -p 5432`
```

Comment:
By port forwarding first? Port forwarding step not mentioned anywhere.

Reply:
i will add all details